src/main/java/picard/sam/AbstractAlignmentMerger.java

```diff
@@ -156,6 +160,29 @@ void close() {
         }
     }
 
+    public enum UNMAPPING_READ_STRATEGY {
```

Comment:
Why is this all upper case?  Enums are classes, so this should really be `UnmappingReadStrategy`.